presto-main/src/main/java/com/facebook/presto/cost/CoefficientBasedStatsCalculator.java

```diff
@@ -46,15 +46,15 @@
 
 import java.util.Map;
 
-import static com.facebook.presto.cost.PlanNodeCost.UNKNOWN_COST;
+import static com.facebook.presto.cost.PlanNodeStatsEstimate.UNKNOWN_STATS;
```

Comment:
Move (aka redo) "Rename PlanNodeCost to PlanNodeStatsEstimate" commit as first in this PR, otherwise it's unreviewable. Once done, put it into separate PR (for _reviewing_ and merging faster)